api/viewsets.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import viewsets
+from serializers import HostSerializer
+
+from updates.models import Host
+
+
+class HostViewSet(viewsets.ModelViewSet):
```

Comment:
I am not so crazy about this. I was planning the API to show Equipments (hwdoc only entities) in the first place. Why expose puppet Hosts ? I 'll have a better look real soon, this is basically a note to self yet